app/components/FieldString/FieldString.css

```diff
@@ -28,15 +28,76 @@
   padding-top: 6px;
 }
 
-.link-preview {
-  width: 100%;
-  margin-top: 5px;
-  margin-bottom: 5px;
-}
-
 .filter-input {
   border: 1px solid var(--theme-colour-background-shade-4, #e4e4e4) !important;
   border-radius: 4px !important;
   font-size: var(--theme-font-size-small, 13px) !important;
   padding: 8px !important;
-}
\ No newline at end of file
+}
+
+.list-container {
+  position: relative;
+}
+
+.list-item {
+  display: flex;
+  align-items: center;
+  padding: 3px;
+  min-height: 31px;
+  background-color: var(--theme-colour-background, white);
+}
+
+.icon-add,
+.icon-drag {
+  background-position: 50%;
+  background-color: var(--theme-colour-background, white);
+  background-image: url('/_public/images/icon-add.svg');
+  background-size: auto 100%;
+  font-size: 0;
+  width: 24px;
+  height: 21px;
+  display: block;
+  flex-shrink: 0;
+  opacity: 0.2;
+}
+
+.icon-drag {
+  background-image: url('/_public/images/icon-drag.svg');
+  cursor: move;
+}
+
+.list-item:hover .icon-drag,
+.dragged .icon-drag {
+  opacity: 1;
+}
+
+.link-preview {
+  white-space: nowrap;
+  margin-right: 5px;
+}
+
+/* .remove-button,
```

Comment:
Can we remove this, or will it be used later on?

Reply:
I did want to use it later, but realistically, it will not be so soon, so I took it out